aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -60,30 +60,8 @@
   return gen;
 }
 
-void runMPSGraph(
```

Comment:
can we keep this code and add a macro to disable it. If we do run into issues, it will be easy to toggle it.

Reply:
We shouldn't push "experimental code" upstream. If this code doesn't work even in one corner case, we should fix or replace it completely. Compile-time Toggling is messy and the code change again needs to be committed upstream when we toggle (so why not clean up the disabled code in the first place).

Reply:
What we have is experimental code currently. As Saharsh noted earlier there are issues we have with running `hf_Albert` at higher batch size on next OS. I am happy if you want to replace it or fix the issue. But as things stand to improve the performance of the networks, I was thinking we can consider this change, and keep a minimal Compile time toggle to turn it off.